src/styles/cart.css

```diff
@@ -0,0 +1,81 @@
+/*
+    Author: Vrushabh Dhatrak
+    Release: 0.1
+    Description : Cart CSS
+*/
+.cart-container {
+    margin:0;
+    padding:0;
+    display: grid;
+    height: 100vh;
+    grid-template-columns:1fr;
+    
+    grid-template-areas: 
+    "header"
+    "my-cart-section"
+    "cart-main-section"
+    "footer"
+    ;
+}
+/* Header Component For Entire Project */
+.header-container {
+    grid-column: header;
+}
+
+/* Filter Results For Entire Project */
+.my-cart-section {
+    grid-column: my-cart-section;
+    margin: 15rem 0 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+/* cards section For Entire Project */
+.cart-main-section {
+    grid-column:cart-main-section;
+    display: flex;
+    justify-content: center;
+}
+
+
+/* footer section For Entire Project */
+.footer {
+    grid-column:footer;
+}
+/* Totals section  */
+
+.totals-item {
+    clear: both;
+    width: 100%;
+    margin-bottom: 10px;
+}
+
+.price-label {
+    float: left;
```

Comment:
Suggestion - Avoid using float

Reply:
Only one place I used this, on the float I got the better result that's why I used that.